src/features/creators/templates/newPackageTemplate/README.md

```diff
@@ -0,0 +1,44 @@
+# package_name
+
+> TODO: Write a short description of your project and what it does.
+
+## Table of Contents
+
+- [package\_name](#package_name)
+  - [Table of Contents](#table-of-contents)
+  - [How to Run](#how-to-run)
+  - [How to Package](#how-to-package)
+  - [How to Test](#how-to-test)
+  - [Setup for New Contributors](#setup-for-new-contributors)
+  - [License](#license)
+
+## How to Run
+
+> TODO: Describe how to use or run your package, with example commands or code.
+1. `<run_exec> -m <package_name>`
```

Comment:
the run exec is returned as absolute here which we don't want in the readme- need to figure out how to make it relative and if it should be relative to root folder? I would assume thats the goal
<img width="1033" alt="Screenshot 2025-04-22 at 3 33 05 PM" src="https://github.com/user-attachments/assets/833d7415-7385-4a33-883c-1c7ff79ded72" />

Reply:
You would probably want to make this an editable install, that way you can run with just `python -m package_name`